deps/spidershim/src/v8string.cc

```diff
@@ -137,6 +137,60 @@ Local<String> String::NewFromTwoByte(Isolate* isolate, const uint16_t* data,
                         length).FromMaybe(Local<String>());
 }
 
+struct ExternalStringFinalizer : JSStringFinalizer {
```

Comment:
> Nit: I've been trying to avoid adding non-public symbols to the v8 namespace. Please move this to either v8::internal or an anonymous namespace.

Makes sense, I moved it to v8::internal in 5bcd5eb.